programs/steward/src/state/large_bitmask.rs

```diff
@@ -0,0 +1,96 @@
+use anchor_lang::{prelude::Result, zero_copy};
+use borsh::{BorshDeserialize, BorshSerialize};
+
+use crate::{constants::MAX_VALIDATORS, errors::StewardError};
+
+const LARGE_BITMASK_INDEXES: usize = MAX_VALIDATORS * 4;
```

Comment:
would just say 20k rather than associating this w/ MAX_VALIDATORS since the two numbers aren't correlated. Also add a note that we are allocating at this size to handle future growth of ValidatorHistory accounts, at 2800 in June 2024

Reply:
Done